tvbextunicore/unicore_wrapper/unicore_wrapper.py

```diff
@@ -7,6 +7,7 @@
 
 import os
 
+from pyunicore.client import _HBP_REGISTRY_URL
```

Comment:
Depending on this private var might pose some danger....

Reply:
I agree, however for now this seems to be the only reliable way to access sites. We could add in the future a way to configure from GUI the endpoint (thus making our extension more reusable - maybe also add login with credentials) but for now in my opinion this seems fine.